recipes-extended/onie-tools/onie-tools/onie-bisdn-upgrade

```diff
@@ -54,7 +56,10 @@ case $# in
 esac;
 
 if [ "${NEED_CONFIRM}" = true ]; then
-  read -p "Upgrading BISDN Linux to install-source: ${INSTALL_FILE} using Client-IP-cfg: dhcp! Please confirm system reboot: [y/N] " yn
+  echo "Upgrading BISDN Linux from ${INSTALL_FILE}."
+  echo "ONLY FILES CONFIGURED FOR RETENTION WILL BE KEPT!"
+  echo "For additional information, see the man page for onie-bisdn-upgrade(1)."
+  read -p "Please confirm system reboot: [y/N] " yn
```

Comment:
and while at it, maybe rewrite this a bit more natural:

```suggestion
  read -p "Reboot the system and upgrade BISDN Linux? [y/N] " yn
```

Reply:
Done.